<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckPartialWindow</class>
 <widget class="QDockWidget" name="CheckPartialWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>374</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono ExtraLight</family>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>CheckPartial</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout" name="checkpartialgridlayout">
    <property name="margin" stdset="0">
     <number>2</number>
    </property>
    <item row="1" column="1">
     <widget class="QScrollArea" name="masterAreaScroll">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono ExtraLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="masterScrollWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>96</width>
         <height>308</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="masterLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin" stdset="0">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QScrollArea" name="dxcAreaScroll">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono ExtraLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="dxcScrollWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>95</width>
         <height>308</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="dxcLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin" stdset="0">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="qsoAreaScroll">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono ExtraLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="qsoScrollWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>95</width>
         <height>308</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="qsoLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin" stdset="0">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="logListLabel">
      <property name="font">
       <font>
        <family>JetBrains Mono ExtraLight</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Log</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="masterListLabel">
      <property name="font">
       <font>
        <family>JetBrains Mono ExtraLight</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Master</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="telnetListLabel">
      <property name="font">
       <font>
        <family>JetBrains Mono ExtraLight</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Telnet</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dxcAreaScroll</tabstop>
  <tabstop>qsoAreaScroll</tabstop>
  <tabstop>masterAreaScroll</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
